### PR TITLE
chats 1-1 reviews

### PR DESCRIPTION
**Pull request by jo-mut**
fixes [#17252], [#17251], [#17250], [#17249]

### CODE REVIEW

File: src/status_im2/common/bottom_sheet/style.cljs
Comment (OmarBasem) [inline src/status_im2/common/bottom_sheet/style.cljs]: I think this `10` need to be applied for Android only

Comment (jo-mut) [reply src/status_im2/common/bottom_sheet/style.cljs]: I checked with iOS and it seems necessary too to achieve the same design

Comment (alwx) [reply src/status_im2/common/bottom_sheet/style.cljs]: Also looks a bit like a magic number. Maybe some constant needs to be introduced or at least some explanation?

**Review by alwx — CHANGES_REQUESTED** [review]

File: src/status_im2/contexts/chat/messages/drawers/view.cljs
Comment (smohamedjavid) [inline src/status_im2/contexts/chat/messages/drawers/view.cljs:1]: We have issue #16722 to replace the usage of `quo/menu-item` with `quo/action-drawer`. 

The design issue will be resolved if we are done with the above issue.

P..S. There is no `menu-item` component in the Design System and removing it from the codebase is advised.

File: src/status_im2/common/bottom_sheet/style.cljs
Comment (jo-mut) [reply src/status_im2/common/bottom_sheet/style.cljs]: > Also looks a bit like a magic number. Maybe some constant needs to be introduced or at least some explanation?

@alwx  I have added a comment on the code to explain how we arrive at that value. The selected item can only have a max-height of the remaining part of the screen not occupied by the bottom-sheet. We arrive at getting by: `window-height ` minus (`bottom-sheet-height` + `(:top insets)`  + `margin-bottom`). We minus the `margin-bottom` as its part of the height not occupied by the bottom-sheet 

```clojure
(- window-height sheet-height top bottom-margin)
```

File: src/status_im2/contexts/chat/messages/drawers/view.cljs
Comment (jo-mut) [reply src/status_im2/contexts/chat/messages/drawers/view.cljs:1]: @smohamedjavid thanks for pointing this out. Is it okay with you if this can be done in another pr. so atleast this can be merged first

File: src/status_im2/common/bottom_sheet/style.cljs
Comment (ajayesivan) [inline src/status_im2/common/bottom_sheet/style.cljs]: How about using a variable for `bottom` instead of hardcoding `8` in both spots? This way, we won't need this comment, and it'll keep our UI safe in case we tweak `bottom` down the road.

**Review by ajayesivan — APPROVED** [review]

File: src/status_im2/contexts/chat/messages/drawers/view.cljs
Comment (smohamedjavid) [reply src/status_im2/contexts/chat/messages/drawers/view.cljs:1]: Sure 👍

**Comment by jo-mut** [comment]
@status-im/mobile-qa  when you have time this is ready

**Comment by pavloburykh** [comment]
> @status-im/mobile-qa when you have time this is ready

@jo-mut hello! Thank you for the PR.

Could you please update description of the issues [#17252], [#17251], [#17250], [#17249] ? It would be helpful in terms of testing to see screenshots of Actual and Expected behaviour. 

Also, I see that there are unresolved requested changes. Could you please make requested changes or mark conversation as resolved in case changes were already made and approved? Thank you!

**Comment by jo-mut** [comment]
@pavloburykh I have updated the description of those issues. Thank you

**Comment by pavloburykh** [comment]
> burykh I have updated the description of those issues. Thank you

Thank you @jo-mut! 

What about this part:

> Also, I see that there are unresolved requested changes. Could you please make requested changes or mark conversation as resolved in case changes were already made and approved? Thank you!

chats 1-1 reviews by jo-mut · Pull Request #17253 · status-im:status-mobile 2023-09-26 17-31-52

**Comment by jo-mut** [comment]
@alwx looks like your approval is necessary to unlock this pr. for merge. Could you have a look at the updates on the requested changes

**Review by alwx — APPROVED** [review]

**Comment by alwx** [comment]
@jo-mut looks good, thanks!

**Comment by pavloburykh** [comment]
Thanx for the PR @jo-mut. 

I have checked the PR and frankly speaking I was not able to see any difference compared to nightly in terms of described areas of fixes.

Let's ask @Francesca-G for review.

**Comment by Francesca-G** [comment]
@pavloburykh we're currently working on the 1.25 build review and we have a lot of comments for 1-1 chats. That should be all ready by Monday, so should we wait for that big review to be ready or does it make sense to review this now? :)

**Comment by pavloburykh** [comment]
> @pavloburykh we're currently working on the 1.25 build review and we have a lot of comments for 1-1 chats. That should be all ready by Monday, so should we wait for that big review to be ready or does it make sense to review this now? :)

Thank you @Francesca-G! This PR provides some fixes based on previous feedback (1.24 release review). 

So I would appreciate if you provide feedback on this PR exclusively in terms of these issues [#17252], [#17251], [#17250], [#17249] 

Based on 1.25 review we will create new umbrella issues which will be fixed in separate PRs. Thank you!

**Review by Francesca-G — COMMENTED** [review]
We still have some issues with margins [[17252](https://github.com/status-im/status-mobile/issues/17252)] as commented [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS/Feedback-for-Mobile?type=design&node-id=6051%3A57375&mode=design&t=PqN01meInlPIXgF0-1)

The rest looks good to me :)

**Comment by pavloburykh** [comment]
Thank you @Francesca-G! 

@jo-mut could you please take another look at https://github.com/status-im/status-mobile/issues/17252 ?

**Comment by jo-mut** [comment]
> Thank you @Francesca-G!
> 
> @jo-mut could you please take another look at #17252 ?

@Francesca-G @pavloburykh I have addressed the design feedback on the margins. Take a look. Thank you

**Comment by pavloburykh** [comment]
> @Francesca-G @pavloburykh I have addressed the design feedback on the margins. Take a look. Thank you

Once @Francesca-G approves the fix PR is ready for merge @jo-mut

**Review by Francesca-G — COMMENTED** [review]
Here's the [review](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS/Feedback-for-Mobile?type=design&node-id=6051%3A57375&mode=design&t=cf6XAwavgbnOhVOW-1) :)

Please refer to review 2.

**Comment by jo-mut** [comment]
> Here's the [review](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS/Feedback-for-Mobile?type=design&node-id=6051%3A57375&mode=design&t=cf6XAwavgbnOhVOW-1) :)
> 
> Please refer to review 2.

@Francesca-G thanks for these comments. I have addressed them if you could have another look just to confirm

**Review by Francesca-G — APPROVED** [review]

**Pull request merged**
